GOVERNANCE.md

```diff
@@ -0,0 +1,111 @@
+* [Keycloak Node.js Connect](https://github.com/keycloak/keycloak-nodejs-connect) - Node.js adapter for Keycloak
+* [Keycloak Node.js Admin Client](https://github.com/keycloak/keycloak-nodejs-admin-client) - Node.js library for Keycloak Admin REST API
+
+The same governance model applies to all projects. However, the list of maintainers may vary per project. 
+
+
+## Maintainers
+
+The list of maintainers can be found in the [MAINTAINERS.md](MAINTAINERS.md) file in the repository for the individual 
+projects listed in the [Projects](#projects) section.
+
+### Maintainer Responsibilities
+
+A maintainer is someone that has shown deep knowledge of vision, features and codebase. It is their 
+responsibility to drive the project forward, encourage collaboration and contributions, and generally help the 
+community.
+
+Responsibilities of a maintainer includes, but are not limited to:
+
+* Engage in design discussions
+* Actively monitor mailing lists, user forum and chat
+* Contribute high quality code
+* Maintain deep knowledge of vision, features and codebase
+* Review pull requests either themselves or delegate to experts in the relevant area
+* Helping the community
+
+### Becoming a Maintainer
+
+To become a maintainer you need to demonstrate the following:
+
+* Good understanding of vision, features and codebase
+* Contribution of larger features
+* Contribution of bug fixes
+* Participation in design discussions
+* Participation in pull request reviews
+* Ability to collaborate with the team
+* Helping the community
+
+A new maintainer must be proposed by sending an email to the [developer mailing list](https://lists.jboss.org/mailman/listinfo/keycloak-dev).
+The email should include evidence of the above list.
+
+The existing maintainers will then discuss the proposal. If anyone objects, or wants more information, the maintainers 
+will reach out to the nominee directly for further discussion. After a review of the nomination existing maintainers
+will vote. The nominee will be accepted as a new maintainer by a 2/3 majority vote.
+
+
+### Changes in Maintainership
+
+Maintainers can be removed by a 2/3 majority vote by maintainers.
+
+
+## Contributing Changes
+
+The process of reviewing proposed changes differs depending of the size and impact of the change.
+
+### Minor Changes
+
+A minor change is a bug fix, a smaller enhancement or a smaller addition to existing features.
+
+To propose a minor change simply create an issue in our [issue tracker](https://issues.jboss.org/browse/KEYCLOAK) and
+send a pull request.
+
+A maintainer will be responsible for ultimately approving the pull request. The maintainer may do a deep review of the
+pull request themselves or delegate to an expert in the corresponding area.
+
+If the maintainer believes the change has a bigger impact it has to follow the process for larger changes.
```

Comment:
I would suggest "... bigger impact, follow the process for larger changes."

Reply:
Will rework this sentence a bit